set-postgres-permissions.bash

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+CI_AZURE_FOLDER="${HOME}/.azure-${AZURE_SUBSCRIPTION_SHORT_NAME}"
+
+if [ -d "${CI_AZURE_FOLDER}" ]
+then
+    echo "Overriding AZURE_CONFIG_DIR to ${CI_AZURE_FOLDER}"
+    export AZURE_CONFIG_DIR="${CI_AZURE_FOLDER}"
+else
+    echo "Directory ${HOME}/.azure-${AZURE_SUBSCRIPTION_SHORT_NAME} does not exist."
```

Comment:
Is that all? No `Warning:`, no `Error:`?

Reply:
no this is designed to run on azure devops too which doesn't do that weird folder thing